### PR TITLE
Lesson 03 task

### PR DESCRIPTION
**Pull request by EkBratchik**

### CODE REVIEW

File: lesson_03/main.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func AverageArray(a [6]float32) float32 {
+	var c float32
```
Comment (dimonl) [inline lesson_03/main.go]: all internal func shouldn't be exported

File: lesson_03/main.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	var k float32
+	k = 0
+	for _, v := range a {
+		c += v
+		k++
+	}
+	return c / k
+}
+func Max(a []string) string {
+	var longestWord string
```
Comment (dimonl) [inline lesson_03/main.go]: all internal func shouldn't be exported

File: lesson_03/main.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+		if length > longestLength {
+			longestWord = word
+			longestLength = length
+		}
+	}
+	return longestWord
+}
+func Reverse(a []int64) []int64 {
+	for i, j := 0, len(a)-1; i < j; i, j = i+1, j-1 {
```
Comment (dimonl) [inline lesson_03/main.go]: same

File: lesson_03/main.go [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+		a[i], a[j] = a[j], a[i]
+	}
+	return a
+}
+func PrintSorted(a map[int]string) {
```
Comment (dimonl) [inline lesson_03/main.go]: need to fix like upper comment

**Review by dimonl — APPROVED** [review]

**Review by rat6666 — APPROVED** [review]

**Pull request merged**